src/sentinel.c

```diff
@@ -4227,6 +4326,86 @@ void sentinelRoleCommand(client *c) {
     dictReleaseIterator(di);
 }
 
+/* SENTINEL GET <mastername> <option> */
```

Comment:
@hwware I don't see the `SENTINEL GET` command implementation mentioned anywhere in the top comment or documentation. Also, I think the way it is implemented is a ambiguous: parameters that are treated either as master name or as a parameter name, depending on the server state, are confusing.

Reply:
Hello @yossigo , sorry but the sentinel get code was from a previous commit from this PR that had to be removed since it is replaced by the commands in the top comment. I have removed that part. Let me know if there is anything which needs to be updated. Thanks!